src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1820,6 +1838,8 @@ private bool IsInTypeofExpression(int position)
                     return SymbolInfo.None;
             }
 
+            boundExpr = AdjustLowestBoundNodeForFunctionPointerConversion(boundExpr);
```

Comment:
>boundExpr = AdjustLowestBoundNodeForFunctionPointerConversion(boundExpr); [](start = 12, length = 73)

It feels like for a successful method group to FP conversion we should be able to get the best candidate that was picked. Are we able to achieve this? It looks like we are always downgrading to a method group here.

Reply:
Yes. The highest `BoundNode` will be a `BoundConversion`, which will have the correct method selected if overload resolution was successful, and that will be used for the "successful" lookup below.

---
In reply to: [430729125](https://github.com/dotnet/roslyn/pull/44436#discussion_r430729125) [](ancestors = 430729125)

Reply:
Based on what we do for invocations, I'll remove special handling of this case.

---
In reply to: [430732394](https://github.com/dotnet/roslyn/pull/44436#discussion_r430732394) [](ancestors = 430732394,430729125)